_1327/documents/templates/documents_edit.html

```diff
@@ -52,6 +55,12 @@ <h2>{% trans "Preview" %} <a id="shareText" class="hidden">Share</a></h2>
 						{% trans "Error! Could not upload image." %}
 					</div>
 					<select class="form-control" id="attachmentModalSelect"></select>
+					<div class="editor-image-modal-scale-container pt-2">
```

Comment:
please fix the indentation in the following lines (and use only tabs)

Reply:
and since you're on it: please fix all the indentations in this file - boy scout rule ;)